rules-tests/DowngradePhp70/Rector/Instanceof_/DowngradeInstanceofThrowableRector/Fixture/nested.php.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Instanceof_\DowngradeInstanceofThrowableRector\Fixture;
+
+use Throwable;
+
+return something() && $e instanceof Throwable;
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\DowngradePhp70\Rector\Instanceof_\DowngradeInstanceofThrowableRector\Fixture;
+
+use Throwable;
+
+return something() && (($throwable = $e) instanceof Throwable || $throwable instanceof \Exception);
```

Comment:
Please add 1 more fixture when before code is already processed as above to handle already processed